app/controllers/spotlight/catalog_controller.rb

```diff
@@ -51,7 +51,10 @@ def autocomplete
   def admin
     self.blacklight_config.view.select! { |k,v| k == :admin_table }
     self.blacklight_config.view.admin_table.partials = [:index_compact]
-    self.blacklight_config.sort_fields.reject! { |k,v| true }
+
+    unless self.blacklight_config.sort_fields.has_key? :timestamp
+      self.blacklight_config.add_sort_field :timestamp, sort: "#{blacklight_config.index.timestamp_field} desc"
```

Comment:
I think that still may have this issue because the previous and next link will be rendered under the context of the app's `CatalogController` and it will attempt to lookup a sort configuration (from the session's searches), which wouldn't have a timestamp sort configured. 

Unless you mean the `AdminController` either blacklists the sort key from being added to the session or just doesn't store its searches at all. (or some other way that I'm sure I'm missing)